ui-tests/src/test/resources/features/customizations/connectors/apicurio.feature

```diff
@@ -135,19 +148,23 @@ Feature: Customization - API Connector - ApicurIO GUI
     And select "Kie Server API" connection type
     Then check that connection authentication type has 1 option and contains text "API Key Authentication"
 
+#    BROKEN in Syndesis 1.8.1-20190918
+  @disabled #to be replaced with Jira issue annotation, when ready
```

Comment:
The Jira annotations are already in place, so please have a look and update accordingly, thanks!

Reply:
ok.